GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2.cfg

```diff
@@ -1,157 +1,807 @@
-//	Leonardo Permanent Multipurpose Module (PMM)
-//	Diameter: 4.57m
-//	Length: 6.6m
-//	Mass: 4.437944128t
-//	Source: https://en.wikipedia.org/wiki/Leonardo_(ISS_module)
 
-@PART[ht2_MPLM]:FOR[RealismOverhaul]
+@PART[ht2_handrail]:FOR[RealismOverhaul]
 {
-	%RSSROConfig = True
-	
-	@MODEL
+	%RSSROConfig = False
+	@rescaleFactor = 1.5625
+	@title = ISS EVA/Service handrail
+	@manufacturer = NASA
+	@description = Handrails serve as mobility and stability aids for astronauts during extravehicular activity. Handrails are installed to define a safe path and provide sturdy anchors for the astronauts to use while they are working outside the spacecraft.
+	@mass = 0.001
+}
+
+@PART[ht2_CBM]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = False
+	!rescaleFactor{}
+	%rescaleFactor = 1.5625
```

Comment:
syntax here is questionable, !node{} is syntax for deleting a single node, not a key/value.
What you want is `!rescaleFactor,* = DEL` to delete all previous ones (hack for the original part maker did it wrong)

Reply:
yeah I wasn't sure how to write it as the original files had 2 rescaleFactor instructions. In order for the config to work correctly, I needed to delete one of the instructions.